src/main/java/com/appoptics/integrations/kafka/broker/Reporter.java

```diff
@@ -64,8 +64,12 @@ public void run() {
             for (PostResult r : result.results) {
                 if (r.isError()) {
                     if (r.response != null) {
-                        LOG.error(String.format("error attempting to post measurements to librato, response code %s",
-                                r.response.getResponseCode()), r.exception);
+                        String errMsg = String.format(
+                                "error attempting to post measurements to librato, response code %s, response body %s",
```

Comment:
might as well make it say appoptics here :)

Reply:
I was planning on a subsequent PR since there are a lot more than just these references